validator/validator_test.go

```diff
@@ -486,8 +487,15 @@ testdata/invalid_validation_localized_message.proto:54:26: message must always r
 			if err != nil {
 				t.Fatal(err)
 			}
-			actual := "\n" + validator.Format(v.Validate(ctx, file))
-			if diff := cmp.Diff(actual, test.expected); diff != "" {
+			got := validator.Format(v.Validate(ctx, file))
```

Comment:
I want to use the same test table to test if we don't receive any errors, so I've slightly updated it...